seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json

```diff
@@ -68,6 +68,42 @@
       "translations": {}
     },
     {
+      "id": "587d7b83367417b2b2512b37",
+      "title": "Clarifying the In-Browser Console Log",
+      "description": [
+        "In an ordinary Javascript file loaded and run in your browser a <code>console.log()</code> will print exactly what you tell it to print to the console. In your in-browser text editor the process is slightly different and can be confusing at first. Any value passed to a <code>console.log</code> runs each set of test data as well as your function call.",
```

Comment:
If someone hasn't read the issue thread, they may not understand why this challenge is here. I think we can re-work this first paragraph as follows to make it clear:

You may have noticed that some freeCodeCamp JavaScript challenges include their own console. This behaves a little differently than the browser console you used in the last challenge.

When you load and run an ordinary JavaScript file in your browser, the `console.log()` statements will print exactly what you tell it to print to the browser console. However, the freeCodeCamp console [... rework that last sentence "Any value passed..." here to clarify exactly what fCC code and console do. I also didn't understand what 'function call' was referring to.]

Reply:
Hmm... a little bit unclear what you are referring to when you say  "include their own console", it's not that the challenges include their own console, it's that the browser console behaves different based on how the code is read...  Maybe I am misunderstanding though...  I'll work this in and see what you think, if I can make git squash work this time around.  😄